manager/allocator/doc.go

```diff
@@ -3,15 +3,15 @@
 // manages a set of independent allocator processes which can mostly
 // execute concurrently with only a minimal need for coordination.
 //
-// One of the instances where it needs coordination is when to move a
-// task to ALLOCATED state. Since a task can move to ALLOCATED state
-// only when all task allocators have completed their service of
-// allocation, they all have to agree on that. The way this achieved
-// in `allocator` is by creating a `taskBallot` to which all task
+// One of the instances where it needs coordination is when deciding to
+// move a task to the ALLOCATED state. Since a task can move to the
+// ALLOCATED state only when all task allocators have completed the allocation
+// of the service, they must cooperate. The way `allocator`
```

Comment:
Task allocation is separate from service allocation. I think case I think the comment was talking about allocation of tasks.